frontend/src/containers/UserListing.tsx

```diff
@@ -9,7 +9,8 @@ import BackendService from "../services/BackendService";
 import { useHistory } from "react-router-dom";
 import Modal from "../components/Modal";
 import AlertContainer from "./AlertContainer";
-import Tooltip from "../components/Tooltip";
+import { MenuItem } from "@reach/menu-button";
+import DropdownMenu from "../components/DropdownMenu";
```

Comment:
@jenboydclark suggestion: We should encapsulate reach/menu-button into the `<DropdownMenu />` component itself. The fact that we need to import both DropdownMenu and MenuItem everytime we need to use this component is going to make our application tightly coupled with the reach/menu-button library, which we should avoid. 

An easy way to do this would be to create a subcomponent attached to the <DropDownMenu> component. So when we use it it would look something like this: 

```js

const MenuItem = DropdownMenu.MenuItem;

<DropdownMenu>
  <MenuItem>Action 1</MenuItem>
</DropdownMenu>
```